src/test/java/io/reactivex/internal/operators/single/SingleDematerializeTest.java

```diff
@@ -18,62 +18,90 @@
 import io.reactivex.*;
 import io.reactivex.exceptions.TestException;
 import io.reactivex.functions.Function;
+import io.reactivex.internal.functions.Functions;
 import io.reactivex.subjects.SingleSubject;
 
 public class SingleDematerializeTest {
 
     @Test
     public void success() {
         Single.just(Notification.createOnNext(1))
-        .<Integer>dematerialize()
+        .dematerialize(Functions.<Notification<Integer>>identity())
```

Comment:
Too bad the identity function is inside an internal package :(

Reply:
I suppose most users now can use lambdas in their project so this is just `v -> v` for them. Only we are still so unlucky to use such helper methods.